.flowconfig

```diff
@@ -99,5 +99,4 @@ unsafe-getters-setters
 ;untyped-type-import
 sketchy-null
 
-[version]
```

Comment:
You think this is useless? It's true that I always need to change this file when I upgrade flow... and the rest of the config is left untouched anyway.

Reply:
Yes, I think it is, I noticed it is also removed from `incubator/universe`. 
Well actually the tag is there, but no version specified. Which is basically the same.